config/src/main/java/org/springframework/security/config/web/server/ServerHttpSecurity.java

```diff
@@ -1682,6 +1683,16 @@ public AuthorizeExchangeSpec authenticated() {
 				return access(AuthenticatedReactiveAuthorizationManager.authenticated());
 			}
 
+			/**
+			 * Require a specific IP Address or subnet
+			 * @param ipAddressExpression the ipaddress (i.e. 192.168.20.199) or local
+			 * subnet
```

Comment:
Perhaps `the address or range of addresses from which the request must come`?

Reply:
Done.